mcstatus/pinger.py

```diff
@@ -147,124 +147,5 @@ async def test_ping(self) -> float:
-                self.sample = [PingResponse.Players.Player(p) for p in raw["sample"]]
-            else:
-                self.sample = None
-
-    class Version:
-        name: str
-        protocol: int
-
-        def __init__(self, raw):
-            if not isinstance(raw, dict):
-                raise ValueError(f"Invalid version object (expected dict, found {type(raw)})")
-
-            if "name" not in raw:
-                raise ValueError("Invalid version object (no 'name' value)")
-            if not isinstance(raw["name"], str):
-                raise ValueError(f"Invalid version object (expected 'name' to be str, was {type(raw['name'])})")
-            self.name = raw["name"]
-
-            if "protocol" not in raw:
-                raise ValueError("Invalid version object (no 'protocol' value)")
-            if not isinstance(raw["protocol"], int):
-                raise ValueError(f"Invalid version object (expected 'protocol' to be int, was {type(raw['protocol'])})")
-            self.protocol = raw["protocol"]
-
-    players: Players
-    version: Version
-    description: str
-    favicon: Optional[str]
-    latency: float = 0
-
-    def __init__(self, raw):
-        self.raw = raw
-
-        if "players" not in raw:
-            raise ValueError("Invalid status object (no 'players' value)")
-        self.players = PingResponse.Players(raw["players"])
-
-        if "version" not in raw:
-            raise ValueError("Invalid status object (no 'version' value)")
-        self.version = PingResponse.Version(raw["version"])
-
-        if "description" not in raw:
-            raise ValueError("Invalid status object (no 'description' value)")
-        self.description = self._parse_description(raw["description"])
-
-        self.favicon = raw.get("favicon")
-
-    @staticmethod
-    def _parse_description(raw_description: Union[dict, list, str]) -> str:
-        if isinstance(raw_description, str):
-            return raw_description
-
-        if isinstance(raw_description, dict):
-            entries = raw_description.get("extra", [])
-            end = raw_description["text"]
-        else:
-            entries = raw_description
-            end = ""
-
-        description = ""
-
-        for entry in entries:
-            for style_key, style_val in STYLE_MAP.items():
-                if entry.get(style_key):
-                    try:
-                        if isinstance(style_val, dict):
-                            style_val = style_val[entry[style_key]]
-
-                        description += f"§{style_val}"
-                    except KeyError:
-                        pass  # ignoring these key errors strips out html color codes
-            description += entry.get("text", "")
-
-        return description + end
+class PingResponse(JavaStatusResponse):
+    """Outdated interface for `JavaStatusResponse`."""
```

Comment:
We probably don't need to define another child class, let's just set this deprecated version to the imported deprecated one.
```suggestion
PingResponse = JavaStatusResponse
```

Reply:
Though again, I'd probably be better to simply move the entire class into this file, which if agreed on, obviously renders this request not applicable and irrelevant